core/arch/arm/mm/tee_mmu.c

```diff
@@ -952,7 +958,7 @@ TEE_Result tee_mmu_map_param(struct user_mode_ctx *uctx,
 		    param_type != TEE_PARAM_TYPE_MEMREF_OUTPUT &&
 		    param_type != TEE_PARAM_TYPE_MEMREF_INOUT)
 			continue;
-		if (param->u[n].mem.size == 0)
+		if (param->u[n].mem.mobj == NULL)
```

Comment:
To be somewhat consistent with other code in OP-TEE.
```suggestion
		if (!param->u[n].mem.mobj)
```

Reply:
done